modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/openapi/contributor/ObjectEntryOpenAPIContributor.java

```diff
@@ -213,6 +243,15 @@ private void _addObjectRelationshipSchema(
 				_bundleContext, getExternalDTOClassName(objectDefinition),
 				_openAPIResource);
 		}
+		else if (GetterUtil.getBoolean(
```

Comment:
I don't understand this if clause compared with the else. What if the feature flag is not active? the schema is not being associated properly? In that case, I guess we are fixing a bug, and for that case, maybe we don't need to add a feature flag?